Formula/p/pcaudiolib.rb

```diff
@@ -0,0 +1,56 @@
+class Pcaudiolib < Formula
+  desc "Portable C Audio Library"
+  homepage "https://github.com/espeak-ng/pcaudiolib"
+  license "GPL-3.0-or-later"
+
+  stable do
+    url "https://github.com/espeak-ng/pcaudiolib/releases/download/1.2/pcaudiolib-1.2.tar.gz"
+    sha256 "6fae11e87425482acbb12c4e001282d329be097074573060f893349255d3664b"
+
+    # Fix -flat_namespace being used on Big Sur and later.
+    patch do
+      url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff"
+      sha256 "35acd6aebc19843f1a2b3a63e880baceb0f5278ab1ace661e57a502d9d78c93c"
+    end
+  end
+
+  head do
+    url "https://github.com/espeak-ng/pcaudiolib.git", branch: "master"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
+  on_linux do
+    depends_on "pkg-config" => :build
```

Comment:
we can also move this to a general build dep rather than linux-specific

Reply:
It should only be used on linux to find brew's `pulseaudio` and `alsa-lib`, e.g.  https://github.com/espeak-ng/pcaudiolib/blob/master/configure.ac#L34

One macOS, it uses CoreAudio, so doesn't need to look up anything.

Reply:
👍